src/main/java/teamrtg/rtg/client/GuiCustomizeWorldScreenRTG.java

```diff
@@ -0,0 +1,815 @@
+package teamrtg.rtg.client;
+
+import java.io.IOException;
+import java.util.Random;
+
+import com.google.common.base.Predicate;
+import com.google.common.primitives.Floats;
+import net.minecraft.client.gui.Gui;
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.client.gui.GuiCreateWorld;
+import net.minecraft.client.gui.GuiListButton;
+import net.minecraft.client.gui.GuiPageButtonList;
+import net.minecraft.client.gui.GuiScreen;
+import net.minecraft.client.gui.GuiSlider;
+import net.minecraft.client.gui.GuiTextField;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.Tessellator;
+import net.minecraft.client.renderer.VertexBuffer;
+import net.minecraft.client.renderer.vertex.DefaultVertexFormats;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.init.Biomes;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.world.biome.Biome;
+import net.minecraft.world.gen.ChunkProviderSettings;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+/**
+ * @author topisani
+ */
+@SideOnly(Side.CLIENT)
+public class GuiCustomizeWorldScreenRTG extends GuiScreen implements GuiSlider.FormatHelper, GuiPageButtonList.GuiResponder {
+    protected String title = "Customize World Settings";
+    protected String subtitle = "Page 1 of 3";
+    protected String pageTitle = "Basic Settings";
```

Comment:
All strings in this class will eventually all need to be localised and the .lang files updated.